src/index.html

```diff
@@ -1,19 +1,41 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
-          user-scalable=no,
-          initial-scale=1.0,
-          maximum-scale=1.0,
-          minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-
-<script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="item-card" data-qa="card">
+      <img class="image-card" src="images/imac.jpeg" alt="Apple imac" width="160px" height="134px">
+      <div class="card-info">
+        <p class="card-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+        <p class="card-code">Код товара: 195434</p>
+        <div class="card-rate">
+          <div class="card-sters">
+            <img src="images/star-active.svg" class="star-active" alt="star-active" width="13px" height="15px">
+            <img src="images/star-active.svg" class="star-active" alt="star-active" width="13px" height="15px">
+            <img src="images/star-active.svg" class="star-active" alt="star-active" width="13px" height="15px">
+            <img src="images/star-active.svg" class="star-active" alt="star-active" width="13px" height="15px">
+            <img src="images/star.svg" class="star" alt="star-active" width="13px" height="15px">
```

Comment:
Not a logic image, implement it only with css and `background` property.

Reply:
I don't understand how, we haven't learned yet